server/methods/accounts/accounts.js

```diff
@@ -238,101 +237,91 @@ Meteor.methods({
       });
 
-      if (!user) {
-        userId = Accounts.createUser({
-          email: email,
-          username: name
-        });
-        user = Meteor.users.findOne(userId);
-        if (!user) {
-          throw new Error("Can't find user");
-        }
-        token = Random.id();
-        Meteor.users.update(userId, {
-          $set: {
-            "services.password.reset": {
-              token: token,
-              email: email,
-              when: new Date()
-            }
-          }
-        });
-        SSR.compileTemplate("accounts/inviteShopMember", ReactionEmailTemplate("accounts/inviteShopMember"));
-        try {
-          return Email.send({
-            to: email,
-            from: `${shop.name} <${shop.emails[0].address}>`,
-            subject: `You have been invited to join ${shop.name}`,
-            html: SSR.render("accounts/inviteShopMember", {
-              homepage: Meteor.absoluteUrl(),
-              shop: shop,
-              currentUserName: currentUserName,
-              invitedUserName: name,
-              url: Accounts.urls.enrollAccount(token)
-            })
-          });
-        } catch (_error) {
-          throw new Meteor.Error(403, "Unable to send invitation email.");
-        }
-      } else {
-        SSR.compileTemplate("accounts/inviteShopMember", ReactionEmailTemplate("accounts/inviteShopMember"));
-        try {
-          return Email.send({
-            to: email,
-            from: `${shop.name} <${shop.emails[0].address}>`,
-            subject: `You have been invited to join the ${shop.name}`,
-            html: SSR.render("accounts/inviteShopMember", {
-              homepage: Meteor.absoluteUrl(),
-              shop: shop,
-              currentUserName: currentUserName,
-              invitedUserName: name,
-              url: Meteor.absoluteUrl()
-            })
-          });
-        } catch (_error) {
-          throw new Meteor.Error(403, "Unable to send invitation email.");
-        }
+      const newUser = Meteor.users.findOne(userId);
+
+      if (!newUser) {
+        throw new Error("Can't find user");
       }
+
+      const token = Random.id();
+
+      Meteor.users.update(userId, {
+        $set: {
+          "services.password.reset": { token, email, when: new Date() }
+        }
+      });
+
+      Reaction.Email.send({
+        to: email,
+        from: `${shop.name} <${shop.emails[0].address}>`,
+        subject: `You have been invited to join ${shop.name}`,
+        html: SSR.render("accounts/inviteShopMember", {
+          homepage: Meteor.absoluteUrl(),
+          shop,
+          currentUserName,
+          invitedUserName: name,
+          url: Accounts.urls.enrollAccount(token)
+        })
+      });
     } else {
-      throw new Meteor.Error(403, "Access denied");
+      Reaction.Email.send({
+        to: email,
+        from: `${shop.name} <${shop.emails[0].address}>`,
+        subject: `You have been invited to join ${shop.name}`,
```

Comment:
These need to be i18n strings, and will also need to be editable in email template editor.
This work was in progress in:
https://github.com/reactioncommerce/reaction/pull/1203
https://github.com/reactioncommerce/reaction/pull/1136